crane-cloud/frontend
# new app dashboard

# Description
a figma design was made of how a new application dashboard should look
like.
Here I try to implement it with currently available data
## Type of change

## Trello Ticket ID
https://trello.com/c/a0bKRyTB
## How Can This Been Tested?
check this branch out and try testing all the edited pages for any bugs. Then check through the code to confirm I followed the required principles.
awaiting on feedback

## Checklist:


# Screenshots
before
![dash bod befor](https://user-images.githubusercontent.com/69305400/131748204-05559419-8d8c-4362-9032-0339cf41731c.png)
after
![new app dashbod](https://user-images.githubusercontent.com/69305400/131748246-b1fd483f-4722-4a89-ac98-e1dfea95f9e7.png)
